Sources/Commands/SwiftPackageTool.swift

```diff
@@ -42,13 +42,18 @@ public class SwiftPackageTool: SwiftTool<PackageToolOptions> {
             seeAlso: type(of: self).otherToolNames()
         )
     }
+    
```

Comment:
nit: extra diff

Reply:
I added since I saw some classes with a space before runImpl(), but now I see it is not consistent. 
Can I make it consistent by removing (or adding) the line on the 4 files that override it ?